misc/examples.css

```diff
@@ -87,3 +87,38 @@ hr {
 .actions-label {
   margin-top: 5px;
 }
+
+.examples-container {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.examples-container .row {
+  margin-top: 10px;
+}
+
+.examples-title-lg {
+  font-size: 24px;
+  font-weight: 600;
+}
+
+.examples-title-sm {
+  margin-left: 10px;
+  font-size: 16px;
+  font-weight: 200;
```

Comment:
So I don't believe this font-weights map to anything. Just 300,400,600,700 and 800